platform/server/http/HttpServer.ts

```diff
@@ -1,42 +1,89 @@
 import * as express from 'express';
-import * as bodyParser from 'body-parser';
 import * as http from 'http';
+import * as https from 'https';
+import { promisify } from 'util';
+import { readFileSync } from 'fs';
+
+import { HttpConfig } from './HttpConfig';
+import { Env } from '../../config';
+import { Logger } from '../../logging';
 import { Router } from './Router';
 
 export class HttpServer {
   private readonly app: express.Application;
-  private readonly httpServer: http.Server;
+  private server: http.Server | https.Server | null = null;
 
-  constructor() {
+  constructor(private readonly log: Logger, private readonly env: Env) {
     this.app = express();
-    this.app.use([
+
+    // We will register body parser only for routes defined in the new platform.
+    /*this.app.use([
```

Comment:
It looks like we don't have to register body parsers "globally" and we can do that on route basis (maybe even router level). I'll check.

Reply:
++ I like the idea of adding them on routers instead